packages/perseus/src/widgets/cs-program/cs-program-validator.ts

```diff
@@ -0,0 +1,29 @@
+import type {PerseusScore} from "../../types";
+import type {PerseusCSProgramUserInput} from "../../validation.types";
+
+export function csProgramValidator(
+    state: PerseusCSProgramUserInput,
+): PerseusScore {
```

Comment:
`rubric` was not being used, so it was removed as a parameter.

Reply:
This is one of those sus things we're going to have to figure out in the future. Can you check to make sure there's a ticket for this? Something along the lines on "some widgets don't use rubric for validation"? I imagine all the program/iframe widgets do this.

Reply:
Here's the relevant ticket: https://khanacademy.atlassian.net/browse/LEMS-2440